ResearchGate.js

```diff
@@ -123,13 +133,13 @@ function scrape(doc, url) {
 	ZU.doGet(risURL, function (text) {
 		// Z.debug(text);
 		// fix wrong itemType information in RIS
-		if (type == "bookSection") text = text.replace('TY  - BOOK', 'TY  - CHAP');
-		if (type == "conferencePaper") text = text.replace('TY  - BOOK', 'TY  - CONF');
+		if (type == "bookSection") text = text.replace('TY	- BOOK', 'TY	- CHAP');
```

Comment:
This spacing seems unusual for RIS -- did something go wrong here during pretty-printing or does RG really reliably use tabs between field code and hyphen?

Reply:
This is probably from me trying to make the linter happy in Vim. Then I guess the linter was hapyy but it messed it up too. Will fix tomorrow.